package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Making company travel and accomodation easy and convinient",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "mocha src/modules/user/tests --recursive --require @babel/register",
```

Comment:
Are we to put test files in each module or use the general test folder in /src/test